action/evmtx_container.go

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2024 IoTeX Foundation
+// This source code is provided 'as is' and no warranties are given as to title or non-infringement, merchantability
+// or fitness for purpose and, to the extent permitted by law, all liability for your use of the code is disclaimed.
+// This source code is governed by Apache License 2.0 that can be found in the LICENSE file.
+
+package action
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/iotexproject/go-pkgs/hash"
+	"github.com/iotexproject/iotex-address/address"
+	"github.com/iotexproject/iotex-proto/golang/iotextypes"
+	"github.com/pkg/errors"
+)
+
+var (
+	_ EvmTxContainer = (*evmTxContainer)(nil)
+)
+
+type evmTxContainer struct {
+	chainID      uint32
+	evmNetworkID uint32
```

Comment:
may not need to be carried by container

Reply:
it is needed when calling `Unfold` to convert the tx